### PR TITLE
Added steps on how to incrementally integrate the wrapper to the phone

### PR DESCRIPTION
**Pull request by erikhagglund**
@emilwestergren review

### CODE REVIEW

File: content/android/gettingstarted.md [hunk 1]
```diff
@@ -167,6 +167,41 @@ Every time you have rebuilt the wrapper library with `build` you can use the `pu
 # Restoring phone
 If you want to reset the device to its original state you can use the `restore` function. This will set the camera library on the device to the original.
 
+# Integrating the android wrapper
+
+The integration procedure is divided into several steps where each step is intended to add more functionality and performance.
+In the Android.mk file there are several build time flags that can be used to configure how the integration is done.
```
Comment (emilwestergren) [inline content/android/gettingstarted.md]: `build flags` is enough

File: content/android/gettingstarted.md [hunk 2]
```diff
@@ -167,6 +167,41 @@ Every time you have rebuilt the wrapper library with `build` you can use the `pu
...
+
+**NOTE:** In order to get the script to build properly when changing flags, remove the build result each time a flag is changed before building.
```
Comment (emilwestergren) [inline content/android/gettingstarted.md]: This is a problem...

Comment (emilwestergren) [inline content/android/gettingstarted.md]: Do you think we should add this to the `build` function?

File: content/android/gettingstarted.md [hunk 3]
```diff
@@ -167,6 +167,41 @@ Every time you have rebuilt the wrapper library with `build` you can use the `pu
...
+    ``` mk
+    LOCAL_CFLAGS += -DPUREWRAPPER=0
+    LOCAL_CFLAGS += -DSINGLEBUFFER=0
+    ```
+
+4. And as a final step the other configuration and feature flags can be used depending on your given platform and HAL version.
+
 # Using the Vidhance API
 
 Examine the *CameraWrapper* implementation in the HAL folder and use it as an example for how to interact with the Vidhance API for Android. Here is a more detailed description of the code using the Vidhance API:
```
Comment (emilwestergren) [inline content/android/gettingstarted.md]: This needs to update in a separate PR.

**Comment by erikhagglund** [comment]
updated.

**Comment by emilwestergren** [comment]
Looks good

**Pull request merged**